### PR TITLE
app-arch/lz4: version bump to 1.8.0

### PR DESCRIPTION
**Pull request by amadio**

### CODE REVIEW

**Comment by amadio** [comment]
I also have a question. `sci-physics/root` will need xxHash. Should we package that separately? Should we change LZ4 config to bundle it?

**Comment by mgorny** [comment]
I dare say separately is better.

File: app-arch/lz4/lz4-1.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+else
+	SRC_URI="https://github.com/lz4/lz4/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Extremely Fast Compression algorithm"
+HOMEPAGE="https://github.com/lz4/lz4"
+
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
```
Comment (mgorny) [inline app-arch/lz4/lz4-1.8.0.ebuild]: Changing subslot to `0` here is certainly wrong. According to [ABI tracker](https://abi-laboratory.pro/tracker/timeline/lz4/), 1.8.0 is not ABI compatible with 1.7.5, so SONAME should be bumped. However, given that the only change was removing stupid `const` from freeing function, I suppose we can ignore that change.

File: app-arch/lz4/lz4-1.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DBUILD_STATIC_LIBS=$(usex static-libs)
+	)
+
+	cmake-utils_src_configure
+
+	# fix missing version in .pc, #608144
+	sed -i -e "/Version/s:$:${PV}:" "${BUILD_DIR}"/liblz4.pc || die
```
Comment (mgorny) [inline app-arch/lz4/lz4-1.8.0.ebuild]: This is clearly no longer necessary.

**Comment by amadio** [comment]
Hi @mgorny, I also prefer separate, obviously. I only asked because xxhash is not in Gentoo yet and is only used internally (no headers or libraries remain at the end after install).

Thanks for picking this up.

**Pull request closed without merging**